pkg/lib/registry/registry.go

```diff
@@ -91,6 +91,9 @@ func (r RegistryUpdater) AddToRegistry(request AddToRegistryRequest) error {
 		simpleRefs = append(simpleRefs, image.SimpleReference(ref))
 	}
 
+	if request.Mode == "" {
+		request.Mode = registry.ReplacesMode
+	}
```

Comment:
This is required due to the change from an integer-based AddMode type to a string based type. Previously `ReplacesMode` value was 0, so the default value of request.Mode was `0`/`ReplacesMode`

Now, the default value is `""`, so we need to check for that case and translate to `ReplacesMode`